utils/postgres_writer.py

```diff
@@ -38,11 +38,11 @@ def insert_postgres_table_if_rows_not_exist(
         )
 
         # Create a SQL INSERT statement with ON CONFLICT DO NOTHING clause
-        insert_sql = f"INSERT INTO {table_name} ({column_names}) VALUES {data_to_insert} ON CONFLICT ({', '.join(unique_key_columns)}) DO NOTHING;"
+        insert_sql = f"INSERT INTO {table_name} ({column_names}) VALUES %s ON CONFLICT ({', '.join(unique_key_columns)}) DO NOTHING;"
 
         # Execute the SQL statement with parameter binding
         with engine.begin() as conn:
-            conn.execute(text(insert_sql))
+            conn.execute(text(insert_sql), data_to_insert)
```

Comment:
I want to see how this behaves. May be it will break...

Reply:
@anayanapalli Tested this locally, but it does break the code ('Error: (psycopg2.errors.SyntaxError) syntax error at or near "%"')